app/views/index.erb

```diff
@@ -0,0 +1,20 @@
+<% if current_user %>
+  <div id="new-link">
+    <form action="/peeps" method="post">
+      <h3>Write your peep here...</h3>   
+      <input type="text" name="content">
+      <input type="submit" value="Peep!">
+    </form>
+  </div>
+<% end %>  
+
+<h2>Peep feed!</h2>
+
+<ul>
+    <% if @peeps %>
+      <% @peeps.reverse.each do |peep| %>
```

Comment:
You've sneaked a bit of logic into your views here - you could instead have encapsulated this inside a class method inside of your peep model (where the logic belongs), maybe having a method like `Peep.all_for_feed` or something